sycl/doc/extensions/Reduction/Reduction_status.md

```diff
@@ -0,0 +1,189 @@
+# Reductions: current implementation and known problems/TODOs
+
+**NOTE**: This document is a quick draft. It is written to help developers of SYCL headers/library to understand the current status, currently used algorithms and known problems.
+
+
+
+# Reduction specifications
+
+There are 2 specifications of the reduction feature and both are still actual:
+
+* `sycl::ext::oneapi::reduction` is described in [this document](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/Reduction/Reduction.md). This extension was created as part of a pathfinding/prototyping work before it was added to SYCL 2020 standard.
```

Comment:
```suggestion
* `sycl::ext::oneapi::reduction` is described in [this document](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/Reduction/Reduction.md). This extension is deprecated, and was created as part of a pathfinding/prototyping work before it was added to SYCL 2020 standard.
```

Reply:
Thank you. Fixed as suggested: 55c3d7537ae5a3ac89729ee4fb88cad41873ab5d